test/_init_tests.js

```diff
@@ -188,6 +188,10 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
+    delete global.sandbox;
```

Comment:
I'm worried about unrelated tests breaking because of this. If these aren't properly cleaned (or the non-global sandboxes above), the next test now has polluted state. Explicitly, I want this:

```js
  const forgotGlobal = !!global.sandbox;
  if (forgotGlobal) {
    delete global.sandbox;
  }
  if (sandboxes.length > 0) {
    sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
    if (forgotGlobal) {
      throw new Error('You forgot to clear global sandbox!');
    }
    throw new Error('You forgot to restore your sandbox!');
  }